client/src/components/sources/sources.js

```diff
@@ -148,13 +152,37 @@ class Sources extends React.Component {
     });
   }
 
-  addSource() {
+  showAddSourceWizard() {
+    this.setState({ addSourceWizardShown: true });
+
+    /*
     Store.dispatch({
       type: toastNotificationTypes.TOAST_ADD,
       alertType: 'error',
       header: 'NYI',
       message: 'Adding sources is not yet implemented'
     });
+    */
+  }
+
+  quitAddSourceWizard() {
+    let heading = <h3>Are you sure you want to quit adding a source?</h3>;
```

Comment:
cancel rather than quit

Reply:
I'd love to use the "cancel" wording, but unless we change the name of the "cancel" button, I think we should be careful on that one.

Reply:
Per @serenamarie125 @bclarhk we're aiming to use ```no``` and ```yes``` for the buttons